functions/api/v2/instance.ts

```diff
@@ -19,7 +19,7 @@ export async function handleRequest(domain: string, db: Database, env: Env) {
 	const res: InstanceConfigV2 = {
 		domain,
 		title: env.INSTANCE_TITLE,
-		version: getVersion(),
+		version: getFederationUA(domain),
```

Comment:
Is this change fixing something? That would be a breaking change and I don't see a reason to do it

Reply:
I think I misunderstood what you were asking for in your last review when you asked why I was suggesting changes to the `getVersion()` method instead of using the `getFederationUA(domain: string)` method. I just reverted this change, and the related change to `wildebeest/config/versions.ts`. I *think* that will yield what you're for, but if I still didn't get it right can you please suggest specific changes or offer an example to clarify the question?